addons/l10n_it_edi/models/account_move.py

```diff
@@ -0,0 +1,1212 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from base64 import b64encode
+from datetime import datetime
+import logging
+from lxml import etree
+from markupsafe import escape
+import uuid
+
+from odoo import _, api, Command, fields, models
+from odoo.addons.base.models.ir_qweb_fields import Markup, nl2br, nl2br_enclose
+from odoo.addons.account_edi_proxy_client.models.account_edi_proxy_user import AccountEdiProxyError
+from odoo.exceptions import UserError
+from odoo.tools import float_compare, float_repr, cleanup_xml_node
+
+_logger = logging.getLogger(__name__)
+
+
+WAITING_STATES = ['processing', 'forward_attempt']
```

Comment:
Don't make a constant variable just for that. It's only use in 2 places.
Use immutable stucture (tuple here) for that instead. It's more efficient for python.